boofuzz/sessions.py

```diff
@@ -1507,8 +1562,7 @@ def _fuzz_current_case(self, path):
             self.transmit_fuzz(target, self.fuzz_node, path[-1], callback_data=callback_data)
 
             if not self._check_for_passively_detected_failures(target=target):
-                self._post_send(target)
-                self._check_procmon_failures(target=target)
+                self._check_for_passively_detected_failures(target=target)
```

Comment:
Why check the return value and then call it again?
This results in post test case callbacks being called twice.

Reply:
I believe that this is the same behavior that is discussed in #229. Because the post-send-callbacks are now monitors, this results in this if fun() fun() thing.